### PR TITLE
1436790/1438462: Part 50: Final 8 files with MSDN link changes.

### PR DESCRIPTION
**Pull request by DennisLee-DennisLee**
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

These are changes to instances I missed or needed to reconsider. There are a few more MSDN links, but most of them are to MSDN magazine articles that aren't on docs.microsoft.com.

### CODE REVIEW

File: docs/framework/wcf/feature-details/serializing-in-json-with-message-level-programming.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ms.assetid: 5f940ba2-57ee-4c49-a779-957c5e7e71fa
...
+ The <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> is used when the <xref:System.ServiceModel.Web.WebGetAttribute> or the <xref:System.ServiceModel.Web.WebInvokeAttribute> is applied to a service operation. Both of these attributes allow you to specify the `RequestFormat` and `ResponseFormat`. To use JSON for requests and responses set both of these to `WebMessageFormat.Json`.  In order to use JSON you must use the <xref:System.ServiceModel.WebHttpBinding> which automatically configures the <xref:System.ServiceModel.Description.WebHttpBehavior>. For more information about WCF serialization, see [Serialization and Deserialization](../../../../docs/framework/wcf/feature-details/serialization-and-deserialization.md). For more information about JSON and WCF, see [Service Station - An Introduction To RESTful Services With WCF](https://msdn.microsoft.com/magazine/dd315413.aspx) and [Creating JSON-enabled WCF Services in .NET 3.5](https://www.pluralsight-training.net/community/blogs/fritz/archive/2008/01/31/50121.aspx).
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/serializing-in-json-with-message-level-programming.md]:
```suggestion
 The <xref:System.Runtime.Serialization.Json.DataContractJsonSerializer> is used when the <xref:System.ServiceModel.Web.WebGetAttribute> or the <xref:System.ServiceModel.Web.WebInvokeAttribute> is applied to a service operation. Both of these attributes allow you to specify the `RequestFormat` and `ResponseFormat`. To use JSON for requests and responses. set both of these to `WebMessageFormat.Json`.  In order to use JSON, you must use the <xref:System.ServiceModel.WebHttpBinding>, which automatically configures the <xref:System.ServiceModel.Description.WebHttpBehavior>. For more information about WCF serialization, see [Serialization and Deserialization](../../../../docs/framework/wcf/feature-details/serialization-and-deserialization.md). For more information about JSON and WCF, see [Service Station - An Introduction To RESTful Services With WCF](https://msdn.microsoft.com/magazine/dd315413.aspx).  
```
The second link appears to be broken.

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ You can see in the image above some example values based on the ViewModel types
   [*https://github.com/StackExchange/dapper-dot-net*](https://github.com/StackExchange/dapper-dot-net)
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md]:
```suggestion
 <https://github.com/StackExchange/dapper-dot-net>
```

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ You can see in the image above some example values based on the ViewModel types
...
 
 - **Julie Lerman. Data Points - Dapper, Entity Framework and Hybrid Apps (MSDN Mag. article)** \
-  [*https://msdn.microsoft.com/magazine/mt703432.aspx*](https://msdn.microsoft.com/magazine/mt703432.aspx)
+  <https://msdn.microsoft.com/magazine/mt703432.aspx>
 
 - **ASP.NET Core Web API Help Pages using Swagger** \
   [*https://docs.microsoft.com/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio*](https://docs.microsoft.com/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio)
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md]:
```suggestion
  <https://docs.microsoft.com/aspnet/core/tutorials/web-api-help-pages-using-swagger?tabs=visual-studio>
```

File: docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/net-core-microservice-domain-model.md [hunk 4]
```diff
@@ -165,8 +165,8 @@ For example, in the preceding OrderAggregate code example, there are several pri
 - **Vaughn Vernon. Modeling Aggregates with DDD and Entity Framework.** Note that this is *not* Entity Framework Core. \
   <https://kalele.io/blog-posts/modeling-aggregates-with-ddd-and-entity-framework/>
 
-- **Julie Lerman. Coding for Domain-Driven Design: Tips for Data-Focused Devs** \
-  [*https://msdn.microsoft.com/magazine/dn342868.aspx*](https://msdn.microsoft.com/magazine/dn342868.aspx)
+- **Julie Lerman. Data Points - Coding for Domain-Driven Design: Tips for Data-Focused Devs** \
+  <https://msdn.microsoft.com/magazine/dn342868.aspx>
 
 - **Udi Dahan. How to create fully encapsulated Domain Models** \
   [*http://udidahan.com/2008/02/29/how-to-create-fully-encapsulated-domain-models/*](http://udidahan.com/2008/02/29/how-to-create-fully-encapsulated-domain-models/)
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/net-core-microservice-domain-model.md]:
```suggestion
  <http://udidahan.com/2008/02/29/how-to-create-fully-encapsulated-domain-models/>
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for all of the work you've put into making these link changes, @DennisLee-DennisLee. I've left some suggestions, almost all unrelated to your PR.

**Comment by rpetrusha** [comment]
//cc @mvelosop for visibility

File: docs/framework/performance/index.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ If you want to create apps with great performance, you should design and plan fo
 |[!INCLUDE[win8_appname_long](../../../includes/win8-appname-long-md.md)] apps written in C++, C#, and Visual Basic|[Performance best practices for Windows Store apps using C++, C#, and Visual Basic](https://docs.microsoft.com/previous-versions/windows/apps/hh750313%28v=win.10%29)|  
 |Windows Presentation Foundation (WPF)|[WPF Performance Suite](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/aa969767(v=vs.100))|  
 |ASP.NET|[ASP.NET Performance Overview](https://docs.microsoft.com/previous-versions/aspnet/cc668225(v=vs.100))|  
-|Windows Forms|[Practical Tips for Boosting the Performance of Windows Forms Apps](https://msdn.microsoft.com/magazine/cc163630.aspx)|  
+|Windows Forms|[Winning Forms: Practical Tips for Boosting the Performance of Windows Forms Apps](https://web.archive.org/web/20060411224813/https://msdn.microsoft.com/msdnmag/issues/06/03/WindowsFormsPerformance/)|
```
Comment (mairaw) [inline docs/framework/performance/index.md]: I'd just remove this 

```suggestion
```

File: docs/framework/winforms/windows-forms-overview.md [hunk 6]
```diff
@@ -61,7 +61,7 @@ For step-by-step information about how to use these features, see the following
 |-----------------|----------------|
 |Using the <xref:System.Windows.Forms.BindingSource> component|[How to: Bind Windows Forms Controls with the BindingSource Component Using the Designer](../../../docs/framework/winforms/controls/bind-wf-controls-with-the-bindingsource.md)|
 |Working with [!INCLUDE[vstecado](../../../includes/vstecado-md.md)] data sources|[How to: Sort and Filter ADO.NET Data with the Windows Forms BindingSource Component](../../../docs/framework/winforms/controls/sort-and-filter-ado-net-data-with-wf-bindingsource-component.md)|
-|Using the Data Sources window|[Walkthrough: Displaying Data on a Windows Form](https://msdn.microsoft.com/library/f6e08c2c-c792-43de-adf3-3e52c0100225)|
+|Using the Data Sources window|[Walkthrough: Displaying Data on a Windows Form](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/wwh8ka92(v=vs.120))|
```
Comment (mairaw) [inline docs/framework/winforms/windows-forms-overview.md]: Let's change this to go to https://docs.microsoft.com/visualstudio/data-tools/bind-windows-forms-controls-to-data-in-visual-studio instead

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good @DennisLee-DennisLee. Left a few comments to be addressed.

**Review by mairaw — APPROVED** [review]
Thanks @DennisLee-DennisLee! Last one - woo hoo!

**Comment by rpetrusha** [comment]
Thank you for making the additional changes, @DennisLee-DennisLee. I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Comment by mvelosop** [comment]
Thank you @rpetrusha!

**Pull request merged**